examples/neural_compressor/text-classification/intent-classification/README.md

```diff
@@ -0,0 +1,61 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Intent classification 
+
+## Clinc task
+
+The script [`run_clinc.py`](https://github.com/huggingface/optimum-intel/blob/main/examples/neural_compressor/text-classification/intent-classification/run_clinc.py)
+allows us to apply static quantization approach as well as distillation 
+using the [Intel Neural Compressor ](https://github.com/intel/neural-compressor) library for 
+sequence classification task.
+
+The following example applies post-training static quantization on a distilled MiniLM_L3 fine-tuned on the clinc_oos task.
+
+```bash
+python run_clinc.py \
+    --model_name_or_path SetFit/MiniLM_L3_clinc_oos_plus_distilled \
+    --apply_quantization \
+    --quantization_approach static \
+    --do_train \
+    --do_eval \
+    --verify_loading \
+    --output_dir /tmp/clinc_output
+```
```

Comment:
I have the following error when running this command line : 
`AttributeError: 'dict' object has no attribute 'to_dict'` coming from : [`fx_op_cfgs`](https://github.com/intel/neural-compressor/blob/v1.14.2/neural_compressor/adaptor/pytorch.py#L3208).
with  `torch` `v1.13.0` and `neural-compressor` `v1.14.2`

Reply:
Looks like this issue is caused by torch v1.13.0, I tried torch v1.12.1, it worked fine. Thank you for raising this issue!